tests/__init__.py

```diff
@@ -50,6 +50,35 @@
 CommunicationRequest
 
 
+def associative_backdate(now, backdate):
```

Comment:
Would [commutative](https://en.wikipedia.org/wiki/Commutative_property) be more appropriate jargon? :)